### PR TITLE
Docs issue-4158: add SingleBookLink

### PR DESCRIPTION
**Pull request by jenmalloy**
### Type of change
- Documentation

### Description

Add SingleBookLink

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [X] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/modules/con-address-space.adoc [hunk 1]
```diff
@@ -9,6 +9,19 @@ An address space is a group of addresses that can be accessed through a single c
 
 {ProductName} has two types of address spaces:
 
+ifdef::SingleBookLink[]
 * link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#con-standard-address-space-messaging[Standard]
+endif::SingleBookLink[]
+
+ifndef::SingleBookLink[]
+* link:{BookUrlBase}{BaseProductVersion}/html-single/using_amq_online_on_openshift/#con-standard-address-space-messaging[Standard]
```
Comment (lulf) [inline documentation/modules/con-address-space.adoc]: I don't think this can be applied here since it references red hat product urls.

**Comment by jenmalloy** [comment]
@lulf Changed per our discussion

File: documentation/assemblies/assembly-planning.adoc [hunk 2]
```diff
@@ -1,6 +1,7 @@
 // This assembly is included in the following assemblies:
 //
 // assembly-service-admin-guide.adoc
+// Installing master.adoc
```
Comment (lulf) [inline documentation/assemblies/assembly-planning.adoc]: Should this be removed or modified to match a filename in this repo?

File: documentation/assemblies/assembly-planning.adoc [hunk 3]
```diff
@@ -1,6 +1,7 @@
 // This assembly is included in the following assemblies:
 //
 // assembly-service-admin-guide.adoc
+// {InstallGuide} master.adoc
```
Comment (jenmalloy) [inline documentation/assemblies/assembly-planning.adoc]: @lulf Is this a better approach?

**Comment by jenmalloy** [comment]
@lulf removed for now

**Pull request merged**